src/Silverback.Integration/Messaging/Configuration/SilverbackBuilderWithConnectionToExtensions.cs

```diff
@@ -40,7 +40,7 @@ public static class SilverbackBuilderWithConnectionToExtensions
                 .AddScopedBehavior<OutboundRouterBehavior>()
                 .AddScopedBehavior<ProduceBehavior>()
                 .Services
-                .AddSingleton<IOutboundRoutingConfiguration, OutboundRoutingConfiguration>();
+                .AddSingleton<IOutboundRoutingConfiguration>(new OutboundRoutingConfiguration());
```

Comment:
This is necessary because otherwise `GetSingletonInstance` does not work. By the way: I have the suspicion that `PublishOutboundMessagesToInternalBus` never worked ;-) It uses the same mechanism, but there is no test anywhere that verifies it.